stations.csv

```diff
@@ -64085,3 +64085,125 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
+68240;Brescia Bus Station;brescia-bus-station;;;45.532982;10.214527;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITBRABB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68240;
+68241;Venice Tronchetto Train Station;venice-tronchetto-train-station;;;45.440440499999994;12.3086015;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITVCEVT;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68241;
+68242;Milan Central Station;milan-central-station;;;45.48642897326118;9.204367099999999;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITMILMM;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68242;
+68243;Palermo Bus Station;palermo-bus-station;;;38.10853447522295;13.366929675670356;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITPMOPE;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68243;
+68244;Milan San Donato Bus Station;milan-san-donato-bus-station;;;45.42905;9.25624;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITMILMS;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68244;
+68245;Siena Central Train Station;siena-central-train-station;;;43.331503441734526;11.322707474742241;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITSAYSC;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68245;
+68246;Salerno FlixBus Bus Stop;salerno-flixbus-bus-stop;;;40.674003902154524;14.769761921564069;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITQSRSF;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68246;
+68248;Florence Villa Constanza Bus Station;florence-villa-constanza-bus-station;;;43.75521789996351;11.172225;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITFLRFV;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68248;
+68249;Turin Stura Bus Station;turin-stura-bus-station;;;45.11742161491804;7.71050566640954;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITTRNTS;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68249;
+68252;Parma Bus Station;parma-bus-station;;;44.81098963;10.32775;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITPAMPB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68252;
+68253;Foggia Bus Station;foggia-bus-station;;;41.46594245301294;15.55342779702478;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITFOGFB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68253;
+68254;Rome Termini Train Station;rome-termini-train-station;;;41.898502;12.503761;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITROMRS;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68254;
```

Comment:
I'm not sure if we need to map these precise stations. These stations usually appear as volatile station. Also, it's skipping some ids (68249 - 68252), I think you'll need to update manually the `stations.distribusion.csv` with only the stations you want create / update before continue with the normal flow to avoid to skip ids.